app/views/items/_item_form.html.erb

```diff
@@ -1,6 +1,5 @@
-<p> <%= flash[:error] %> </p>
 <p>
-  <%= form_for @item, html: { class: 'form-inline' } do |f| %>
+  <%= form_for Item.new, html: { class: 'form-inline' } do |f| %>
```

Comment:
I know that you were told to take the instance variables out of your controller.  But this is actually worse than having extra instance variables.  If you want to discuss why, I'm happy to.